docs/sphinx/user_guide/feature/sort.rst

```diff
@@ -0,0 +1,159 @@
+.. ##
+.. ## Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+.. ## and other RAJA project contributors. See the RAJA/COPYRIGHT file
+.. ## for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+.. ##
+
+.. _sort-label:
+
+================
+Sorts
+================
+
+RAJA provides portable parallel sort operations, which are basic
+parallel algorithm building blocks. They are described in this section.
+
+A few important notes:
+
+.. note:: * All RAJA sort operations are in the namespace ``RAJA``.
+          * Each RAJA sort operation is a template on an *execution policy*
+            parameter. The same policy types used for ``RAJA::forall`` methods
+            may be used for RAJA sorts.
+          * RAJA sort operations accept an optional *comparator* argument so
+            users can perform different types of sort operations. If
+            no operator is given, the default is a 'less' operation and
+            the result is **non-decreasing**.
+
+Also:
+
+.. note:: * For sorts using the CUDA back-end, RAJA uses the implementations
+            provided by the NVIDIA cub library. For information please see
+            :ref:`build-external-tpl-label`.
```

Comment:
The reference here isn't working either.
What are the restrictions on what can be referenced?

Reply:
Do you have an underscore character at the start of the label where you want to reference? i.e., '_build-external-tpl-label'

Reply:
Here's a link to the sphinx doc section: https://www.sphinx-doc.org/en/1.7/markup/inline.html

Reply:
Thanks, I'm trying adding a title and actually committing the label.